apps/sms/test/marionette/attachments_test.js

```diff
@@ -50,7 +50,9 @@ marionette('Attachment picking and sending tests', function() {
       client.waitFor(function() {
         return composer.attachButton.enabled();
       });
-      composer.attachButton.tap();
+      // XXX: See bug 1171950. Use click() instead of tap() to
+      // attach button action always triggers in tests.
+      composer.attachButton.click();
```

Comment:
I see that we use ```attachmentButton.tap()``` in several other places [1] and [2]; How do you feel about introducing the following method in lib/messages.js and reuse it in 3 places so that we have that trick only in one place?
```
addAttachment: function() {
  client.waitFor(function() {
    return this.Composer.attachButton.enabled();
  }.bind(this));

  // XXX: See bug 1171950. Use click() instead of tap() to attach button
  // action always triggers in tests.
  this.Composer.attachButton.click();
  
  // This line we can either leave in tests or move here, up to you
  this.selectSystemMenuOption('Messages Activity Caller');
}
```


[1] https://github.com/mozilla-b2g/gaia/blob/830eb9552419fbf5ed743fb085c1cc82f0bcbdde/apps/sms/test/marionette/composer_test.js#L184

[2] https://github.com/mozilla-b2g/gaia/blob/830eb9552419fbf5ed743fb085c1cc82f0bcbdde/apps/sms/test/marionette/drafts_test.js#L57